### PR TITLE
Location of ring source in FusionRingSource class

### LINKED ISSUE #11: FusionRingSource z placement should be controllable by users
Currently, users can control the radius and the coverage of the FusionRingSource.

The z placement is assumed to be zero.

However, users should be able to control this z placement:

```python
class FusionRingSource(openmc.Source):
    """An openmc.Source object with some presets to make it more convenient
    for fusion simulations using a ring source. All attributes can be changed
    after initialization if required. Default isotropic ring source with a Muir
    energy distribution.
    Args:
        radius: the inner radius of the ring source
        start_angle: the start angle of the ring in radians,
        stop_angle: the end angle of the ring in radians,
        temperature: the temperature to use in the Muir distribution in eV,
    """
    def __init__(
        self,
        radius,
        start_angle: float =0.,
        stop_angle: float = 6.28318530718,
        z_placement=0,
        temperature: float = 20000.,
        fuel='DT'
    ):
```

### PR DESCRIPTION
**Pull request by AI-Pranto**
fix #11

### CODE REVIEW

**Comment by RemDelaporteMathurin** [comment]
Hi @AI-Pranto thanks for this!

And thank you for spotting the typos "DT"/"DD".

Any chance you could add a test to catch these lines in the CI?
I think atm the CI only checks DT cases.

**Comment by AI-Pranto** [comment]
@RemDelaporteMathurin Sorry for the late, I've added a test case for  `test_ring_source.py`

File: tests/test_point_source.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
 from openmc import Source
-
+import pytest
 
 def test_creation():
     my_source = FusionPointSource()
     assert isinstance(my_source, Source)
+
+with pytest.raises(ValueError):
+    FusionPointSource(fuel='топливо')
```
Comment (RemDelaporteMathurin) [inline tests/test_point_source.py:11]: These should be seperate funcitons
```suggestion
def test_wrong_fuel():
    with pytest.raises(ValueError):
        FusionPointSource(fuel='топливо')
```

File: tests/test_ring_source.py [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
+import pytest
 
 
 def test_creation():
-    my_source = FusionRingSource(radius=1)
+    my_source = FusionRingSource(radius=1, z_placement=1)
     assert isinstance(my_source, Source)
+
+with pytest.raises(ValueError):
+    FusionRingSource(radius=1, fuel='топливо')
```
Comment (RemDelaporteMathurin) [inline tests/test_ring_source.py:12]:
```suggestion
def test_wrong_fuel():
    with pytest.raises(ValueError):
        FusionRingSource(radius=1, fuel='топливо')
```

File: openmc_plasma_source/ring_source.py [hunk 3]
```diff
@@ -12,13 +12,15 @@ class FusionRingSource(openmc.Source):
         radius: the inner radius of the ring source
         start_angle: the start angle of the ring in radians,
         stop_angle: the end angle of the ring in radians,
+        z_placement: Location of the ring source. Defaults to 0.
```
Comment (RemDelaporteMathurin) [inline openmc_plasma_source/ring_source.py:15]:
```suggestion
        z_placement: Location of the ring source (m). Defaults to 0.
```

**Review by RemDelaporteMathurin — CHANGES_REQUESTED** [review]
Thanks again for this @AI-Pranto 

Only tiny changes required

**Pull request merged**